docs/team/justinyjt.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Justin Yip's Project Portfolio Page
+---
+
+
+Tinner (Anagram of Intern) is a desktop app for managing internship applications, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, Tinner allows you to easily sort through and retrieve the relevant information faster than traditional GUI apps.
+
+Given below are my contributions to the project.
+
+* **Code contributed**:
+    * https://nus-cs2103-ay2122s2.github.io/dashboards/contents/tp-progress.html#:~:text=A0222764_-,7,-Forked
```

Comment:
I think your link is wrong. Should be this
`https://nus-cs2103-ay2122s2.github.io/tp-dashboard/?search=justinyjt&breakdown=true`.